cloudinary-core/src/main/java/com/cloudinary/Uploader.java

```diff
@@ -247,9 +247,18 @@ public Map rename(String fromPublicId, String toPublicId, Map options) throws IO
         params.put("to_public_id", toPublicId);
         params.put("invalidate", ObjectUtils.asBoolean(options.get("invalidate"), false).toString());
         params.put("to_type", options.get("to_type"));
+        setBooleanParam(options, params, "context");
+        setBooleanParam(options, params, "metadata");
         return callApi("rename", params, options, null);
     }
 
+    private void setBooleanParam(Map options, Map<String, Object> params, String key) {
```

Comment:
Please remove this function and treat values in a regular way.

Reply:
done